### PR TITLE
Initial Bot app and GitHub Issues scheduler

### LINKED ISSUE #2: On a reoccurring basis check repositories for new issues.
We can't expect all projects to use our webhook.  For those who don't we should periodically check on their repo for new issues and raise awareness.

Scheduling should be handled within our application with no external dependencies (ie CRON), `Process.send_after/4` should work for our initial version.

### PR DESCRIPTION
**Pull request by doomspork**
This closes #2 and is still a WIP.

### CODE REVIEW

File: apps/bot/config/config.exs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+use Mix.Config
+
+config :bot,
+  github_access_key: System.get_env("GITHUB_ACCESS_KEY")
```
Comment (ybur-yug) [inline apps/bot/config/config.exs]: For deployment do we need to make sure this is `Application.get_env/2`?

Comment (doomspork) [inline apps/bot/config/config.exs]: @ybur-yug what do you mean?  I'm not sure I follow.  You wouldn't call `Application.get_env` inside of your configuration file, what would that do?

I build my releases by injecting my ENVs into a docker container so it compiles with those values populated, saves me the hassle of maintaining the values in my systemd configure.

Comment (ybur-yug) [inline apps/bot/config/config.exs]: Derp. Disregard, sorry.

**Comment by doomspork** [comment]
This should be ready for a review.  I'm going to put the pieces together and give 'er a go locally.

**Comment by doomspork** [comment]
Thanks @ybur-yug 👍

**Pull request merged**